release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2922,7 +2922,9 @@ The Node Tuning Operator can now properly select kernel arguments and management
 
 * Previously, an {ibm-power-server-name} cluster installation failed on installer-provisioned infrastructure because the installation program used a random network type instead of using of the specified network type that was specified in the `install-config.yaml` configuration file. With this release, the installation program now uses the network type that is specified in the `install-config.yaml` so that this issue no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-45484[*OCPBUGS-45484*])
 
-* Previously, the validation of Non-Uniform Memory Access (NUMA) node topologies expected equal core indices. This expectation caused the Performance Profile Creator (PPC) to report inaccurate information for compute nodes. With this release, the validation check no longer expects equal core indices so that the PPC now reports accurate information for compute nodes. (link:https://issues.redhat.com/browse/OCPBUGS-44644[*OCPBUGS-44644*])
+* Previously, the Performance Profile Creator (PPC) failed to build a performance profile for compute nodes that had different core ID numbering, core per socket, for their logical processors and the nodes existed under the same node pool. For example, the PPC fails in a situation for two compute nodes that have logical processors `2` and `18`, where one node groups them as core ID `2` and the other node groups them as core ID `9`.
```

Comment:
`core ID numbering, core per socket, for their logical processors and the nodes existed under the same node pool.`
I'd change a bit like this:
` .. core ID numbering (core per socket) for their logical processors where the nodes existed under the same node pool.`